feat: LSP8MetadataBaseURI ERC725Y JSONSchema key (#152)
lukso-network/LIPs: LSPs/LSP-8-IdentifiableDigitalAsset.md

}
```

#### LSP8MetadataBaseURI

The base URI for the LSP8 tokenIds metadata.<br>
To access the metadata of a token, we need to create 

> a URL following this pattern: `{LSP8MetadataBaseURI}{tokenId}`

Reviewer: I really like how the code block describes the pattern here 👌

> ⚠️ TokenIds needs to be in lowercase (even for address tokenId type)

Reviewer: ```suggestion
⚠️ TokenIds need to be in lowercase (even for the tokenId type `address`)
```

> - TokenId type <b>uint256</b><br>

Reviewer: I suggest to put the tokenId type as code for emphasis.

```suggestion
- TokenId type `uint256`<br>
```

e.g. `http://mybase.uri/1234`

> - TokenId type <b>address</b><br>

Reviewer: ```suggestion
- TokenId type `address`<br>
```

> e.g. `http://mybase.uri/43fb7ab43a3a32f1e2d5326b651bbae713b02429`

Reviewer: should there be the `0x` here before? Since it is an address

```suggestion
e.g. `http://mybase.uri/0x43fb7ab43a3a32f1e2d5326b651bbae713b02429`
```

> - TokenId type <b>bytes32</b><br>

Reviewer: ```suggestion
- TokenId type `bytes32`<br>
```
